js/components/cohort-definition-manager.js

```diff
@@ -562,17 +569,91 @@ define(['knockout', 'text!./cohort-definition-manager.html',
 			self.closeConceptSet();
 		}
 
+		function createConceptSet() {
```

Comment:
Cannot we move this code to smth like ConceptSetService and make methods more abstract?

Reply:
I promise to think about that but let do such refactoring outside from current issue